src/components/mobile/AeAccountReverse.vue

```diff
@@ -34,8 +35,12 @@ export default {
   },
   props: {
     address: { type: String, required: true },
-    name: { type: String, default: '' },
+    source: { type: Object, required: true },
```

Comment:
`sourse` is not used in this component

Reply:
Actually, it is needed to calculate `getName(this)`